browser/src/control/jsdialog/Widget.TreeView.ts

```diff
@@ -101,12 +105,27 @@ class TreeViewControl {
 			if (!data.headers || data.headers.length === 0)
 				L.DomUtil.addClass(this._container, 'ui-treeview-tree');
 		} else this._container.setAttribute('role', 'grid');
+
+		if (data.enabled !== false) {
+			this._container.addEventListener('click', TreeViewControl.onClick);
+		}
 	}
 
 	get Container() {
 		return this._container;
 	}
 
+	static getTreeView(e: any) {
```

Comment:
why do we need this static?

isn't better to just work on current instance?

Reply:
good point, yes, I think it is not needed the static. Thanks.